docs/CONTRIBUTING.md

```diff
@@ -129,9 +129,11 @@ For example: `Iteration::Create` or `ToolingJob::Process`
 
 ## View Components
 
-We use view components to split the UI into stand-alone units that can be used and tested independently from the rest of the application.
+We use View Components to split the UI into stand-alone units that can be used and tested independently from the rest of the application.
 View Components are either functional and written in React, or non-functional and rendered as HAML templates.
 
+Each View Component has a class in Rails which is used to render it in a consistent manner, and through which any initial data or configuration can be passed.
+
 ### React components
```

Comment:
can more be added to this? What does the `react_component` method expect? How does this view component actually load the React Component? How can I view the component as it would appear on the website?

Reply:
If you fancy adding your understanding based on what we're worked through (or even sumarise it here and I can word it if you prefer) that'd be helpful. I'm not sure of the best way to explain it other than going through what we did.